apps/aecore/src/aec_hard_forks.erl

```diff
@@ -70,6 +72,21 @@ ensure_env() ->
 protocol_effective_at_height(Height) ->
     protocol_effective_at_height(Height, protocols()).
 
+-spec protocol_vsn_name(protocol_vsn()) -> atom().
+protocol_vsn_name(?ROMA_PROTOCOL_VSN)    -> roma;
+protocol_vsn_name(?MINERVA_PROTOCOL_VSN) -> minerva;
+protocol_vsn_name(?FORTUNA_PROTOCOL_VSN) -> fortuna;
+protocol_vsn_name(?LIMA_PROTOCOL_VSN)    -> lima;
+protocol_vsn_name(?IRIS_PROTOCOL_VSN)    -> iris.
+
+-spec protocol_vsn(atom()) -> protocol_vsn().
```

Comment:
I see this is being used only in `aecontract_SUITE`, so it could be contained there? My concern is those two functions being used in code instead of tests and we have those names all over the place again. @jur0 worked hard to refactor those :D An alternative approach would be `-ifdef(TEST).`-s if we want to keep those here.

Reply:
If you don't use `protocol_effective_at_height`, I'm ok with that. I'd maybe put those names to the same hrl file as `*_PROTOCOL_VSN` defines.

Reply:
I admit that it should be probably better to have this in separate PR... The idea was just to have a way to translate protocol version to name and back - for better error reporting (which wasn't realized at the end here as the check went out from aeprimop)... I checked with Juraj, it doesn't seem to interfere with his work.